docs/source/quickstart.ipynb

```diff
@@ -148,6 +150,19 @@
     "and use ```$(host `hostname` | awk '{print $NF}')``` instead of `host` in `RADICAL_PILOT_DBURL` environment variable."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### x.2.2. `MPI Implementation library (Optional)`\n",
```

Comment:
similar with MongoDB - remove backquotes - as an option to keep it for "MPI" only, but not for a whole string